samples/cpp/cloning_gui.cpp

```diff
@@ -453,7 +453,7 @@ int main()
 
     for(;;)
     {
-        char key = (char) waitKey(0);
+        int key = waitKey(0) & 0xFF;
 
         if(key == 'd' && flag3 == 0)
```

Comment:
Hmm I get your point and agree, but isn't it best to serve a solution that tends to mass? 
Maybe we should add a notice somewhere covering this special case?